x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_activity_log.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback } from 'react';
+
+import { EuiEmptyPrompt, EuiSpacer } from '@elastic/eui';
+import { TimelineEntry } from './components/timeline_entry';
+import * as i18 from '../translations';
+import { SearchBar } from '../../../../components/search_bar';
+import { Immutable, EndpointAction } from '../../../../../../common/endpoint/types';
+import { AsyncResourceState } from '../../../../state';
+
+export const EndpointActivityLog = memo(
+  ({ endpointActions }: { endpointActions: AsyncResourceState<Immutable<EndpointAction[]>> }) => {
+    // TODO
+    const onSearch = useCallback(() => {}, []);
+    return (
+      <>
+        <EuiSpacer size="l" />
```

Comment:
Just a follow up from my prior comment around using `theme.eui` variables for padding. If the tab stuff can stick to `paddingSizes.xl`, you might be able to adjust the gap between content and tabs by adding some additional `EuiSpacer` here or increasing the size. So that you avoid the `calc()` I suggested earlier

Reply:
Actually, I tried adding additional `EuiSpacer` components here and it didn't work and thus I had to add the hardcoded padding. Let's see how all this unnecessary bit of style fixes can be avoided with the component refactor using [flyout#more-complicated-flyout](https://elastic.github.io/eui/#/layout/flyout#more-complicated-flyout)